packages/vite/src/node/shortcuts.ts

```diff
@@ -112,6 +112,13 @@ const BASE_SHORTCUTS: CLIShortcut[] = [
       openBrowser(url, true, server.config.logger)
     },
   },
+  {
+    key: 'c',
+    description: 'clear console',
+    async action(server) {
+      server.config.logger.clearScreen('silent')
```

Comment:
my bad. looks like `silent` isn't an accepted option. let's go back to `error` then.
```suggestion
   action(server) {
      server.config.logger.clearScreen('error')
```